admin-dev/themes/default/template/controllers/tags/helpers/form/form.tpl

```diff
@@ -64,10 +64,10 @@
 		$('#move_to_left').click(function(){
 			return !$('#select_right option:selected').remove().appendTo('#select_left');
 		});
-		$('#select_left option').live('dblclick', function(){
+		$('#select_left').on('dblclick', 'option', function(e) {
```

Comment:
Prefer this, because if you have a js which change #select_left, the event will not be triggered again.
```suggestion
		$(document).on('dblclick', '#select_left option', function(e) {
```